packages/leva/src/components/UI/JoystickPlayground3d.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { JoystickPlayground } from './StyledJoystick'
+import { JoystickGrid3d, JoystickCube } from './StyledJoystick3d'
+
+export function JoystickPlayground3d({
+  playgroundRef,
+  grabberRef,
+  isOutOfBounds,
+  isKeyPressedForPlane1,
+  isKeyPressedForPlane2,
+  children,
+}: {
+  playgroundRef: React.RefObject<HTMLDivElement>
+  grabberRef: React.RefObject<HTMLSpanElement>
+  isOutOfBounds: boolean
+  isKeyPressedForPlane1: boolean
+  isKeyPressedForPlane2: boolean
+  children: JSX.Element
+}) {
+  return (
+    <JoystickPlayground ref={playgroundRef} isOutOfBounds={isOutOfBounds}>
+      <JoystickCube xz={isKeyPressedForPlane1} zy={isKeyPressedForPlane2}>
+        <JoystickGrid3d className="joy--front" />
+        <JoystickGrid3d className="joy--front-mid" />
+        {/* <JoystickGrid3d className="joy--front-rear" /> */}
+        <JoystickGrid3d className="joy--back" />
```

Comment:
I generally don't leave comments in code, but I like having a full cube too. Happy to remove

Reply:
Now that this is encapsulated as just a cube it feels better, but open for comments